DscResources/MSFT_xMySqlGrant/MSFT_xMySqlGrant.schema.mof

```diff
@@ -2,12 +2,12 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlGrant")] 
 class MSFT_xMySqlGrant : OMI_BaseResource
 { 
-  [Key, Description("Name of MySQL user.")] String UserName;
-  [Key, Description("MySql database name to grant permissions.")] String DatabaseName;
-  [Required, Description("MySql connection credential used for the root."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
-  [Write, Description("MySql user permission type."), ValueMap{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}, Values{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}] String PermissionType;
-  [Write, Description("Ensure given grant to mySql database present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-
+    [Key, Description("Name of MySQL user.")] String UserName;
+    [Key, Description("MySql database name to grant permissions.")] String DatabaseName;
+    [Required, Description("MySql connection credential used for the root."), EmbeddedInstance("MSFT_Credential")] String RootCredential;
+    [Key, Description("MySql user permission type."), ValueMap{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}, Values{"ALL PRIVILEGES", "CREATE", "DROP", "DELETE", "INSERT", "SELECT", "UPDATE", "EXECUTE"}] String PermissionType;
```

Comment:
I made PermissionType a key property because you need to call xMySqlGrant multiple times if you want to grant multiple permission types and you can't put those all in the same config if you use just UserName and DatabaseName as key properties.  Adding PermissionType makes a unique key.
It seemed to me like you should know what version you are working with.  Plus then your config wouldn't be broken when the next revision of the module changes the default version.